whitepaper/typography.tex

```diff
@@ -0,0 +1,28 @@
+\section*{Typographical conventions}
+%\addcontentsline{toc}{section}{Typographical conventions}

```

Comment:
delete ?

Reply:
wasn't sure if it should have entry in TOC or not, but I think not